src/main/scala/gestalt/dotty/DottyToolbox.scala

```diff
@@ -1051,8 +1053,10 @@ class Toolbox(enclosingPosition: Position)(implicit ctx: Context) extends Tbox {
 
     def transform(tree: tpd.Tree)(pf: PartialFunction[tpd.Tree, tpd.Tree]): tpd.Tree = {
       new t.TreeMap() {
-        override def transform(tree: tpd.Tree)(implicit ctx: Context) =
-          pf.lift(tree).getOrElse(super.transform(tree))
+        override def transform(tree: tpd.Tree)(implicit ctx: Context) = {
+          def updateOwner(subtree: tpd.Tree) = ensureOwner(subtree, ctx.owner)
```

Comment:
This is because `ctx` is set up as below in `TreeMap`: https://github.com/lampepfl/dotty/pull/2826